contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,21 @@
-### 🎯 Goal 1: People think of PostHog as a mobile solution
+# 📱Goal 1: People think of PostHog as a mobile solution
 
-* Mobile replay for React Native
-* Launch & charge for mobile replay
-* Lots of content on how to use PostHog on Mobile
+* onboard Yiannis and get him rocketing
+* mobile replay out of beta
+* flutter replay
 
-### 🎯 Goal 2: Error tracking in beta
+# 📹 Goal 2: Replay playback 3000
 
-* the top 5 features implemented
-    * SDKs capture errors in one front-end and one back-end language (almost certainly JS and Python)
-    * Stack traces are actionable and human-readable
-    * Similar errors are grouped together
-    * High-volume errors trigger the Alerting system
-    * Integrated with other products such as Session replay
-* real customers using it and giving us feedback
-* integrated with CDP for notifications
-* integrated with replay for investigation
+* onboard Richard and get him rocketing
+* Replay UX Overview
```

Comment:
Would be cool to have a few subtasks for this or at least a few "goals" where we can read and say yeah indeed it's done, right now it's very generic